scss/components/_buttons.scss

```diff
@@ -285,14 +286,20 @@
 // Styleguide components.buttons.icon-buttons
 
 .button--search {
-  @include u-background-image('i-search--primary');
+  @include u-icon-button($magnifying-glass);
+  background-size: 2.2rem;
```

Comment:
More out of curiosity, why does the background size/position need to be set individually for each? Is it no standardized for each icon?